content/applications/hr/recruitment/time_in_stage.rst

```diff
@@ -0,0 +1,82 @@
+======================
+Time in stage analysis
+======================
+
+The *Time In Stage Analysis* report provides information on how long applicants stay in each stage
+of the recruitment process. This is important, as every job position has specific :ref:`process
+details <recruitment/new_job_position/edit>` that state the length of time applicants should expect
+to wait between specific stages.
+
+Knowing how long applicants remain in each stage can help highlight possible bottlenecks. Analyzing
+this data allows the recruitment team to assess each stage, identify any issues, and pivot their
+strategies to move applicants through each stage, within the expected time interval.
+
+Time in stage analysis report
+=============================
+
+To access the report, navigate to :menuselection:`Recruitment app --> Reporting --> Time in Stage
+Analysis`. By default, the report displays data from all job positions, with the stages populating
+the x-axis, and the number of days populating the y-axis, in a line chart.
+
+The default filter is :guilabel:`Last 365 Days Applicant`, showing information for the last 365
+days.
+
+Hover over a stage in the line chart to reveal a popover window listing all the job positions within
+it, and the average number of days each job position sits in each stage.
+
+.. image:: time_in_stage/time-in-stage.png
+   :align: center
+   :alt: The default Time In Stage Analysis report.
+
+Bar chart view
+--------------
+
+For a more visually digestible view of the information in the :guilabel:`Time In Stage Analysis`
+report, click the :icon:`oi-bar-chart` :guilabel:`(Bar Chart)` icon in the upper-left corner. This
+displays all the information in a bar chart.
+
+In this view, it is easier to visualize the differences between the job positions, regarding how
+long applicants stay in each stage. From this view, recruiters can more easily determine which job
+positions have delays or bottlenecks at certain stages.
+
+.. image:: time_in_stage/bar-chart.png
+   :align: center
+   :alt: The bar chart view of the Time In Stage Analysis report.
```

Comment:
I'd like to see this image retaken with some more zoom and hovering over a bar to show the popover mentioned earlier in the doc :)

For example:
![time-in-stage-bar-chart-fs8](https://github.com/user-attachments/assets/1c032bc9-d508-4dbd-8b8f-65a6d0af6bf8)

```suggestion
.. image:: time_in_stage/bar-chart.png
   :align: center
   :alt: The bar chart view of the Time In Stage Analysis report.
```